src/ScriptCs/ScriptCsArgs.cs

```diff
@@ -56,6 +56,10 @@ public ScriptCsArgs()
         [ArgDescription("Allows installation of packages' prelease versions")]
         public bool AllowPreRelease { get; set; }
 
+        [ArgShortcut("rsl")]
+        [ArgDescription("Refresh the cache of script libraries")]
+        public bool RefreshScriptLibraries { get; set; }
```

Comment:
Yes this was left accidentally as I pulled the new command but forgot to change the args. Fixing now.